default/python/universe_generation/galaxy.py

```diff
@@ -15,19 +15,23 @@ def __init__(self, universe_width):
         self.max_dist = universe_tables.MAX_STARLANE_LENGTH
         self.cell_size = min(max(universe_width / 50, self.min_dist), self.max_dist/sqrt(2))
         self.width = int(universe_width / self.cell_size) + 1
-        self.grid = defaultdict(list)
-        print "Adjacency Grid: width", self.width, ", cell size", self.cell_size
+        self.grid = defaultdict(set)
+        print "Adjacency Grid: width {}, cell size {}".format(self.width, self.cell_size)
 
     def cell(self, pos):
+        """Returns cell index"""
```

Comment:
The docstring is a phrase ending in a period. https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring